components/CheckDocumentation/vacuum/XminHorizon.tsx

```diff
@@ -0,0 +1,342 @@
+      </p>
+      <p>
+        When VACUUM is blocked and dead rows can't be cleaned, it can result to
+        table bloat and slow queries.
+      </p>
+      <h4>{causeTitle}</h4>
+      <ul>
+        <GuidanceByBackend inApp={inApp} heldBackInfo={byBackend} />
+        <GuidanceByReplicationSlot
+          inApp={inApp}
+          heldBackInfo={byReplicationSlot}
+          serverReplicationUrl={serverReplicationUrl}
+        />
+        <GuidanceByReplicationSlotCatalog
+          inApp={inApp}
+          heldBackInfo={byReplicationSlotCatalog}
+          serverReplicationUrl={serverReplicationUrl}
+        />
+        <GuidanceByStandby inApp={inApp} heldBackInfo={byStandby} />
+        <GuidanceByPreparedXact inApp={inApp} heldBackInfo={byPreparedXact} />
+      </ul>
+    </div>
+  );
+};
+
+const GuidanceByBackend: React.FunctionComponent<{
+  inApp: boolean;
+  heldBackInfo: number | null;
+}> = ({ inApp, heldBackInfo }) => {
+  if (inApp && !heldBackInfo) {
+    return null;
+  }
+
+  const CodeBlock = useCodeBlock();
+  return (
+    <li>
+      <h5>Long-running transactions</h5>
+      <p>
+        Long-running transactions may still need to access rows that could
+        otherwise be considered dead, so they can block cleanup.
+      </p>
+      <h6>
+        <b>Solution</b>
+      </h6>
+      {heldBackInfo && (
+        <p>
+          A long running transaction is holding back the xmin horizon at{" "}
+          <code>{heldBackInfo["xmin"]}</code>.
+        </p>
+      )}
+      <p>
+        You can find the transaction holding back the xmin horizon and its
+        connection's pid by running the following command:
+      </p>
+      <CodeBlock>
+        <SQL
+          sql={`SELECT pid, datname, usename, state, backend_xmin, backend_xid
+                  FROM pg_stat_activity
+                  WHERE backend_xmin IS NOT NULL OR backend_xid IS NOT NULL
+                  ORDER BY greatest(age(backend_xmin), age(backend_xid)) DESC;`}
+        />
+      </CodeBlock>
+      <p>You can cancel it by running either of commands:</p>
+      <CodeBlock>
+        <SQL
+          sql={`SELECT pg_cancel_backend('<query_pid>');
+                SELECT pg_terminate_backend('<query_pid>');`}
+        />
+      </CodeBlock>
+    </li>
+  );
+};
+
+const GuidanceByReplicationSlot: React.FunctionComponent<{
+  inApp: boolean;
+  heldBackInfo: number | null;
+  serverReplicationUrl: string;
+}> = ({ inApp, heldBackInfo, serverReplicationUrl }) => {
+  if (inApp && !heldBackInfo) {
+    return null;
+  }
+
+  const Link = useSmartAnchor();
+  const CodeBlock = useCodeBlock();
+  return (
+    <li>
+      <h5>Lagging or stale physical replication slots</h5>
+      <p>
+        With physical streaming replication with{" "}
+        <code>hot_standby_feedback</code> is on, when replication is lagging or
+        a replica server is stale (e.g. down, gone), the oldest transaction that
```

Comment:
```suggestion
        a replica server is stale (e.g. down), the oldest transaction that
```
I think "gone" is pretty vague and not that different from "down". One example is probably enough to give users the general idea here.